src/schema/v2/filterArtworksConnection.ts

```diff
@@ -273,12 +273,24 @@ export const FilterArtworksFields = () => {
       type: new GraphQLList(Artist.type),
       description:
         "Returns a list of merchandisable artists sorted by merch score.",
-      resolve: ({ aggregations }, _options, { artistsLoader }) => {
+      args: {
+        size: {
+          type: GraphQLInt,
+          description: "The number of artists to return",
+          default: 12,
```

Comment:
random, isn't it `defaultValue`(instead of `default`)?

Reply:
Oh lordy — will fix!